task/01-strings-tasks.js

```diff
@@ -267,7 +309,38 @@ function isString(value) {
  *   'K♠' => 51
  */
 function getCardId(value) {
-    throw new Error('Not implemented');
+    let id = 0;
```

Comment:
A simpler option is to use the "indexOf" function from an array of characters since the index is the same as id